content/rule_book/roles_and_responsibilities.md

```diff
@@ -2,68 +2,96 @@
+
+##### Reference Questions
+
+On a Reference type question, if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will ask, "What is your question?"
+
+##### Answer Correction
+
+On a Quote/Finish type question, once the quizzer has quoted to the end of the required verse or verses, if the quote is not yet word-perfect, the quizmaster will say, "Again." This will occur as often as needed until the quizzer quotes the verse perfectly or the question answer duration period expires. The quizmaster is not allowed to say "again" until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
+
+For all other question types, once all of the information in both the question and answer has been given, but something needs to be fixed, the quizmaster will say "Again".
+
+##### Quote is Complete
+
+On a Situation type question, when the quizzer has completed the quotation, the quizmaster will say, "Quote is complete."
+
+##### Appending an Answer
+
+On an INT or MA question, if all the information in the answer text has been given but the quizzer is missing information in the question text, the quizmaster will say "More".
+
+On a Situation question, because the quotation is part of the question, the quizmaster will say "More" when required information from the quotation is missing.
+
+On a Quote or Reference question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say "More".
 
 #### Commentary
-On a CVR/MACVR, in the scenario where the entire Reference (which is part of the "Question") was not read by the quizmaster, the quizmaster has two prompts to give, and must give them in appropriate order and with appropriate timing. If the quizzer provides all information in the question "text" and the answer "text", because the quizzer has not yet finished saying the complete Reference (part of the "question"), the quizmaster must prompt with "More." The quizmaster cannot prompt with "What is your question?" yet, because the quizzer has not provided all information in the "question" (which includes the Reference) and the answer.
+
+On a CVR/MACVR, in the scenario where the entire reference (which is part of the question text) was not read by the quizmaster, the quizmaster has 2 prompts to give, and must give them in appropriate order and with appropriate timing. If the quizzer provides all information in the question text and the answer text, because the quizzer has not yet finished saying the complete reference, which is part of the question, the quizmaster must prompt with "More." The quizmaster cannot prompt with "What is your question?" yet, because the quizzer has not provided all information in the question (which includes the reference) and the answer.
```

Comment:
should we also beef it up and change "appropriate" to "correct?"  "appropriate order / appropriate timing"

Reply:
8225e59